index.html

```diff
@@ -108,11 +108,11 @@ <h1 class="ui header">
   var html = '';
   $.each(response[fields.results], function(index, value) {
     html += '<a class="result" href="'+ value.url +'">'
-         +  '<div class="image"> <img src="/img/'+ value.image +'"></div>'
+         +  '<div class="image"><img src="/img/'+ value.image +'" /></div>'
          +  '<div class="content">'
          +  '<div class="title">' + value.title + '</div>'
          +  '<div class="description">' + value.description + '</div>';
-    $.each(['keywords', 'clients', 'partners'], function(i, key) {
+    $.each(['keywords', 'partners'], function(i, key) {
```

Comment:
What is the purpose of this ?

Reply:
When we added html to the `clients` definition of https://github.com/betagouv/api.gouv.fr/blame/43c68c20a6f94737c43b228418276eb9227b8d42/_api/franceconnect.md#L6
It broke the search preview as such:
<img width="434" alt="screen shot 2019-02-01 at 13 40 58" src="https://user-images.githubusercontent.com/1475946/52123667-25b96d80-2627-11e9-995b-a7362408e0dd.png">